__tests__/index.tests.js

```diff
@@ -39,3 +43,155 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => { return fs.readFile('./endpoints.json', "utf-8").then((endpoints)=>{
+        const endpoints1=JSON.parse(endpoints)
+        
+         
+          return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              
+              expect.arrayContaining(Object.keys(endpoints1)  ))
+            
+            })
+        })
+
+        
+           
+              
+                     
+          })
+      });
+
+    
+    
+    describe('error 404 on GET /api bad path', ()=>{
+      test('GET:404', ()=>{
+          
+              return request(app)
+              .get('/apii')
+              .expect(404)
+     
+          })
+      })
+      describe('GET /api/articles/:article_id', () => {
+        test('GET:200 sends an object with the article corresponding to the id given to the client', () => {
+          
+          return request(app)
+            .get(`/api/articles/1`)
```

Comment:
Agree with August, you should change this to make sure that the id is 1, the title is 'Living in the shadow of a great man', author is 'mitch' or whatever your first article is